src/Razor/test/testapps/ComponentApp/ComponentApp.csproj

```diff
@@ -1,9 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
-  <PropertyGroup>
-    <RazorSdkDirectoryRoot>$(RazorSdkArtifactsDirectory)$(Configuration)\sdk-output\</RazorSdkDirectoryRoot>
```

Comment:
Uneeded?

Reply:
Unneeded and wrong as far as I can tell. I've been operating under the assumption that this was left over from John's move of a bunch of code.